doc/sphinx/proofs/automatic-tactics/auto.rst

```diff
@@ -552,11 +552,13 @@ Creating Hints
          leading `( _* )` to match the tail of the path. (Note that `(_*)`
          misparses since `*)` would end a comment.)
 
-      .. warning::
+      .. knownissue:: Hint Cut regexp precedence
```

Comment:
I used the reused the `admonition` and `warning` HTML classes, so that known issues are highlighted like warnings, but I see your point and I'll switch to the `error` class that gives a slightly different color:

![Screenshot 2021-11-02 at 12-20-28 Programmable proof search — Coq 8 15+alpha documentation](https://user-images.githubusercontent.com/1108325/139837285-48e2e060-235e-413f-9949-47163dcbc86e.png)